lib/diagnostic.rb

```diff
@@ -115,10 +116,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = [:age 22, :first_name gerry]'
```

Comment:
if you do it this way, you need a `=>` inbetween the key and value

`Diagnostic.person_hash = 'person = {age: 22, first_name: 'gerry'}' OR `Diagnostic.person_hash = 'person = {:age => 22, :first_name => 'gerry'}`